tests/klay_blockchain_test.go

```diff
@@ -0,0 +1,271 @@
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package tests
+
+import (
+	"crypto/ecdsa"
+	"github.com/klaytn/klaytn/blockchain"
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/consensus/istanbul"
+	"github.com/klaytn/klaytn/crypto"
+	"github.com/klaytn/klaytn/networks/p2p"
+	"github.com/klaytn/klaytn/node"
+	"github.com/klaytn/klaytn/node/cn"
+	"github.com/klaytn/klaytn/params"
+	"github.com/klaytn/klaytn/ser/rlp"
+	"github.com/klaytn/klaytn/work"
+	"io/ioutil"
+	"math/big"
+	"os"
+	"testing"
+	"time"
+)
+
+// TestSimpleBlockchain
+func TestSimpleBlockchain(t *testing.T) {
+	//if testing.Verbose() {
+	//	enableLog() // Change verbosity level in the function if needed
+	//}
+
+	numAccounts := 12
+	fullNode, node, validator, chainId, workspace := newBlockchain(t)
+	defer os.RemoveAll(workspace)
+
+	// create account
+	richAccount, accounts, contractAccounts := createAccount(t, numAccounts, validator)
+	contractDeployCode := "0x608060405234801561001057600080fd5b506000808190555060646001819055506101848061002f6000396000f300608060405260043610610062576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff16806302e5329e14610067578063197e70e41461009457806349b667d2146100c157806367e0badb146100ec575b600080fd5b34801561007357600080fd5b5061009260048036038101908080359060200190929190505050610117565b005b3480156100a057600080fd5b506100bf60048036038101908080359060200190929190505050610121565b005b3480156100cd57600080fd5b506100d6610145565b6040518082815260200191505060405180910390f35b3480156100f857600080fd5b5061010161014f565b6040518082815260200191505060405180910390f35b8060018190555050565b806000540160008190555060015460005481151561013b57fe5b0660008190555050565b6000600154905090565b600080549050905600a165627a7a72305820ef4e7e564c744de3a36cb74000c35687f7de9ecf1d29abdd3c4bcc66db981c160029"
+	for i := 0; i < numAccounts; i++ {
+		contractAccounts[i].Addr = deployContractDeployTx(t, node.TxPool(), chainId, richAccount, contractDeployCode)
+	}
+	time.Sleep(10 * time.Second)
+
+	// deploy
+	for i := 0; i < numAccounts; i++ {
+		deployRandomTxs(t, node.TxPool(), chainId, richAccount, 10)
+		deployValueTransferTx(t, node.TxPool(), chainId, richAccount, accounts[i%numAccounts])
+		deployContractExecutionTx(t, node.TxPool(), chainId, richAccount, contractAccounts[i%numAccounts].Addr)
+	}
+
```

Comment:
```suggestion
	 time.Sleep(5 * time.Second)
	 
```
Wait until the deployed txs are executed.

Reply:
review reflected :)